pkg/eventpersister/persister/persister.go

```diff
@@ -317,19 +313,6 @@ func (p *Persister) send(messages map[string]*puller.Message) {
 			}
 			evs[id] = eventJSON
```

Comment:
We can delete `evs`, `eventJSON`, and the process of marshal event.

Reply:
I have removed it.
https://github.com/bucketeer-io/bucketeer/pull/192/commits/73d315dca1127282c8a055b0c7a94532ce6d9dd0#diff-ad00cb580cd315d8619de51e511dc06052ace57d6146c30b60366e3a97b56d6cL291-L303